.github/workflows/_deploy_kafka_test.yml

```diff
@@ -55,7 +61,7 @@ jobs:
 
       - name: Run Workload
         env:
-          AZURE_AKV_RESOURCE_ENDPOINT: ${{ secrets.SKR_CLIENT_AKV_ENDPOINT }}
+          AZURE_AKV_RESOURCE_ENDPOINT: ${{ secrets.SKR_CLIENT_AKV_ENDPOINTT }}
```

Comment:
misspelling here

Reply:
not a misspelling, just a separate variable steven created with his test AKV (should probably be named as such instead of adding extra letters for the future)